hack/scripts/ocp-bundle-postprocess.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+#  2024 NVIDIA CORPORATION & AFFILIATES
+#
+#  Licensed under the Apache License, Version 2.0 (the License);
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an AS IS BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set -o nounset
+set -o pipefail
+set -o errexit
+
+if [[ "${TRACE-0}" == "1" ]]; then
+    set -o xtrace
+fi
+
+# Generate relatedImages
+cd hack && $GO run release.go --with-sha256 --templateDir ./templates/related-images/ --outputDir . && cd ..
+cat hack/related_images.yaml >> bundle/manifests/nvidia-network-operator.clusterserviceversion.yaml
+rm hack/related_images.yaml
+# Add containerImage annotation
```

Comment:
This is for this field in the UI
![image](https://github.com/user-attachments/assets/f6112ff4-de9e-45a1-9bc2-3d4c10a4085c)